lib/app-components/addon/components/project-metadata/component.ts

```diff
@@ -23,16 +42,38 @@ export default class ProjectMetadata extends Component {
     @service toast!: Toast;
 
     node!: Node;
+    changeset!: BufferedChangeset;
 
     @requiredAction continue!: () => void;
 
+    init() {
+        super.init();
+        this.changeset = buildChangeset(this.node, nodeValidations);
+    }
+
     @task
     @waitFor
     async reset() {
         this.node.rollbackAttributes();
         await this.node.reload();
     }
 
+    @task
+    @waitFor
+    async save() {
+        this.changeset.validate();
+        if (this.changeset.isValid) {
+            try {
+                await this.changeset.save();
+                this.onSave();
+            } catch (e) {
+                this.onError();
```

Comment:
Would it be worthwhile to send the error message to sentry here, even if you aren't displaying to the user? Or are you expecting not to have a useful error message?